src/main/java/me/totalfreedom/totalfreedommod/command/Command_nickclean.java

```diff
@@ -23,12 +23,39 @@
     @Override
     public boolean run(CommandSender sender, Player playerSender, Command cmd, String commandLabel, String[] args, boolean senderIsConsole)
     {
-        FUtil.adminAction(sender.getName(), "Cleaning all nicknames", false);
-
-        for (final Player player : server.getOnlinePlayers())
+        if (args.length == 0)
```

Comment:
Why not `args.length != 1`, the command only requires one argument (player name or `-a`)